README.md

```diff
@@ -1,7 +1,2 @@
 # Typometer
 Basic GUI app to test your typing speed made on QT framework.
-
-If you get any compilation errors please have a look to : http://wiki.qt.io/Transition_from_Qt_4.x_to_Qt5
```

Comment:
Compilations issues had been obserevd with Qt4.

Reply:
Ah, didn't know you were testing with Qt4 as well, makes sense. Might be a good idea to specify in the readme which qt version and compiler is known to work? I used Qt 5.7 with MSVC2013

Reply:
Maybe you could make a ReadMe file. ;)

Reply:
Sure, Ill do that tonight.